### DISCUSSION TITLE
Redis command result returning

### DISCUSSION THREAD

**Discussion by loctd98** [dbeaver/dbeaver#22959] [Q&A]
When i run command `KEYS *`, the result returning not show value column and i can not edit value in the result. Would anyone know how to resolve this?

DBeaver Version
23.3.0.202312201943
Driver
Redis Java Client 4.4.7

<img width="1099" alt="Screen Shot 2024-02-26 at 21 28 58" src="https://github.com/dbeaver/dbeaver/assets/67491511/3a525fdc-f789-4f8f-af21-a1a4ed7fcd4a">

**Comment by LonwoLonwo** [comment]
Hello @loctd98 

https://redis.io/commands/keys/

Because it is a system list, you can't change it this way. 
If you want to delete the key - use the [delete command](https://redis.io/commands/del/). And [set](https://redis.io/commands/set/) as a create command.

**Reply by loctd98** [reply]
Thank you! Hope you develop the feature convert command to sql soon

**Reply by LonwoLonwo** [reply]
You can create new keys in the database navigator tree like this:

![2024-02-28 13_30_22-](https://github.com/dbeaver/dbeaver/assets/45152336/3a5236a2-4dd7-4477-9ec1-ebed597aebc2)

![2024-02-28 13_30_38-DBeaver Ultimate 24 0 0 - _duck_010 db_ Script-178](https://github.com/dbeaver/dbeaver/assets/45152336/7ea2194a-5777-4811-ac44-16115e2c2a64)
